pkg/dyninst/procmon/process_monitor.go

```diff
@@ -94,11 +95,12 @@ func (pm *ProcessMonitor) NotifyExit(pid uint32) {
 // newProcessMonitor is injectable with a fake FS for tests.
 func newProcessMonitor(h Handler, procFS string, resolver ContainerResolver) *ProcessMonitor {
 	pm := &ProcessMonitor{
-		handler:    h,
-		procfsRoot: procFS,
-		resolver:   resolver,
-		eventsCh:   make(chan event, 1024),
-		doneCh:     make(chan struct{}),
+		handler:            h,
+		procfsRoot:         procFS,
+		resolver:           resolver,
+		eventsCh:           make(chan event, 1024),
+		doneCh:             make(chan struct{}),
+		executableAnalyzer: makeExecutableAnalyzer(128),
```

Comment:
Curious why you picked 128 for the LRU cache size? You might also consider defining these values as consts to document the rationale